### PR TITLE
OCLOMRS-279: Migrate from react-bootstrap to reactstrap

### PR DESCRIPTION
**Pull request by Duncanian**
# JIRA TICKET NAME:
[Migrate from react-bootstrap to reactstrap](https://issues.openmrs.org/browse/OCLOMRS-279)

# Summary:
Migrate from react-bootstrap(which uses bootstrap 3) to reactstrap(which uses bootstrap 4) as the app is mainly using Bootstrap 4.

### CODE REVIEW

File: src/components/dashboard/components/dictionary/common/DictionaryModal.jsx [hunk 1]
```diff
@@ -179,27 +179,24 @@ export class DictionaryModal extends React.Component {
     return (
       <div>
         <Modal
-          show={this.props.show}
+          isOpen={this.props.show}
           onHide={this.props.modalhide}
```
Comment (muhozi) [inline src/components/dashboard/components/dictionary/common/DictionaryModal.jsx]: @Duncanian In reactstrap `onHide` is `onClosed` and `dialogClassName` is `modalClassName`. For more about modal component props check this https://reactstrap.github.io/components/modals/

File: src/components/dashboard/components/dictionary/common/DictionaryModal.jsx [hunk 2]
```diff
@@ -338,7 +332,8 @@ export class DictionaryModal extends React.Component {
                   </FormGroup>
                   <FormGroup style={{ marginTop: '12px' }}>
                     Description{' '}
-                    <FormControl
+                    <Input
+                      type="textarea"
                       componentClass="textarea"
```
Comment (muhozi) [inline src/components/dashboard/components/dictionary/common/DictionaryModal.jsx]: @Duncanian Remove `componentClass` prop, `type`prop is used in reactstrap instead and you used it.

Comment (Duncanian) [reply src/components/dashboard/components/dictionary/common/DictionaryModal.jsx]: I have done it.

File: src/components/dashboard/components/dictionary/common/DictionaryModal.jsx (hunk 1 above)
Comment (Duncanian) [reply src/components/dashboard/components/dictionary/common/DictionaryModal.jsx]: I have done it.

**Pull request merged**